plugins/uberdev/skills/testers-pipeline/aggregate.py

```
@@ -0,0 +1,88 @@
+    args = p.parse_args()
+
+    invariants = {i["id"] for i in yaml.safe_load(open(args.invariants))["invariants"]}
+
+    findings = []
+    cross_refs_in = []
+    follow_ups = {}
+    dispositions = []
+
+    for path in sorted(glob.glob(os.path.join(args.scratch_dir, "*/*.yaml"))):
+        try:
+            doc = yaml.safe_load(open(path)) or {}
```

Ruff (0.15.13)

[warning] Use a context manager for opening files

(SIM115)